pkg/osquery/tables/tablehelpers/getconstraints_test.go

```diff
@@ -101,6 +101,14 @@ func TestGetConstraints(t *testing.T) {
 			expected: []string{"a", "b"},
 			opts:     []GetConstraintOpts{WithDefaults("a", "b"), WithAllowedCharacters("z")},
 		},
+
+		// allowed values
+		{
```

Comment:
is this supposed to be a test for the `WithAllowedValues` option?

Reply:
good catch. Let's see if the tests pass! Editing in PRs, always fun